initial/pom.xml

```diff
@@ -15,6 +15,16 @@
 
     <dependencies>
         <dependency>
+            <groupId>org.springframework.cloud</groupId>
```

Comment:
I'm not really seeing any difference between the `initial/pom.xml` and the `complete/pom.xml`. Is that how it's supposed to be? I would think that the initial would not have the starter in it.